core/src/Cabana_Halo.hpp

```diff
@@ -376,7 +379,8 @@ class Gather<HaloType, AoSoAType,
     void reserve( const HaloType& halo, AoSoAType& aosoa )
     {
         if ( !haloCheckValidSize( halo, aosoa ) )
-            throw std::runtime_error( "AoSoA is the wrong size for gather!" );
+            throw std::runtime_error( "Cabana::gather:reserve (AoSoAType): "
```

Comment:
```suggestion
            throw std::runtime_error( "Cabana::gather:reserve (AoSoA): "
```
(same for slice cases below)